### PR TITLE
Es6 awesome book

### PR DESCRIPTION
**Pull request by tylher**
- Created modules for different functions of the page
- Added date and time to the page using Luxon module
- Made sure add and remove buttons still work the same way
- Added more styles to the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@ <h2 class="logo">Awesome Books</h2>
       </nav>
```
Comment (dasileker) [inline index.html:21]: - [ ]Nice job however the `li` is not switching between links in the `navbar`.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @tylher,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- PR is good 👌.
- Linters are passing 👌.
- Good readme file 👌.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tylher) [reply index.html:21]: Thank you, fixed it already
I did not know it was not working on deployed version

**Review by Lameck1 — APPROVED** [review]
Hi, @tylher,

Tasks for this project have been successfully implemented 💯.
Well done working on the concerns from the previous review 👍🏼.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 
- Linters check ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**